config/sheet_music.py

```diff
@@ -0,0 +1,42 @@
+SOURCE_QUERY = "*:*"
+
+
+def get_id(record: dict) -> str:
+    return record.get("id")
+
+
+def map_record(record: dict) -> dict:
+
+    # only take a selection of fields for now
+    # use the "keyword" version of fields since these are deduped
+    fields_to_keep = [
+        "id",
+        "title_keyword",
+        "publisher_keyword",
+        "subjectTopic_keyword",
+        "nameNamePart_keyword",
+        "url_keyword",
+    ]
+    keep_record = {}
+    for fld in fields_to_keep:
+        if fld in record:
+            keep_record[fld] = record[fld]
+
+    # rename fields for consistency
+    output_record = {}
+    output_record["id"] = keep_record["id"]
+    output_record["title"] = keep_record["title_keyword"]
```

Comment:
These are raising `KeyError` exceptions with most / all records, because most / all records (at least in the first few hundred) are missing one or more of those fields.  There's a lot of output of `'subjectTopic_keyword'
'url_keyword'` etc., because the exception handling in `centralsearch.py` (line 90) is too minimal.  Bonus points for fixing that, at least switching to `print(repr(e))`, since printing a `KeyError` by default just prints the missing key and not the type of error.

A better approach: I think if you use a dictionary of "real_fieldname": "renamed_fieldname", you can avoid having redundant keep_record/output_record and don't have to worry about whether fields exist.  Line 21 would iterate over `fields_to_keep.keys()`; if the real field name is present in the source record, add it to `keep_record` using the desired field name.